<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Podaj_ip_dialog</class>
 <widget class="QDialog" name="Podaj_ip_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#ip_Edit[state = &quot;&quot;]{
	background-color: rgb(46, 47, 48)
}

#ip_Edit[state = &quot;0&quot;]{
	background-color: rgb(255, 90, 93);
}

#ip_Edit[state = &quot;1&quot;]{
	background-color: rgb(68, 255, 35);
}

</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Podaj Adres oraz nazwÄ™ serwera</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>401</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Adres IP:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="ip_Edit">
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
       <property name="state" stdset="0">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nazwa Serwera:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nazwa_serwera_Edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Podaj_ip_dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>391</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>264</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
